test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -261,3 +262,16 @@ def test_tensorflow_distribute_dataservice_gpu(
     if test_utils.is_image_incompatible_with_instance_type(tensorflow_training, ec2_instance_type):
         pytest.skip(f"Image {tensorflow_training} is incompatible with instance type {ec2_instance_type}")
     run_data_service_test(ec2_connection, ec2_instance_ami, tensorflow_training, TF_DATASERVICE_DISTRIBUTE_TEST_CMD)
+
+# Placeholder for habana test
+# Instance type and AMI to be updated once the EC2 Gaudi instance is available
+# Using exisitng instance to run the test
+@pytest.mark.model("N/A")
+#@pytest.mark.parametrize("ec2_instance_type", TF_EC2_HPU_INSTANCE_TYPE, indirect=True)
+#@pytest.mark.parametrize("ec2_instance_ami", [test_utils.HPU_AL2_DLAMI], indirect=True)
```

Comment:
are these commented out parameterizations needed?

Reply:
removed the comments